libs/components/layout/src/lib/modules/box/box.component.html

```diff
@@ -9,7 +9,9 @@
   }"
 >
   <div class="sky-box-header-content">
-    <ng-content select="sky-box-header"></ng-content>
+    <div class="sky-box-header">
```

Comment:
Do we need a `div` with `sky-box-header` here? The element inserted as the `ng-content` will already have a `div` with the `sky-box-header` CSS class, which could be confusing when debugging.